test/e2e/__tests__/index.js

```diff
@@ -67,6 +71,12 @@ describe('E2E Tests', () => {
    */
   exportCsvData()
 
+  /**
+   * Survey template.
+   */
+  templateCreate()
+  templateDelete()
```

Comment:
this must be addeed at the end of everything. before recordDelete

Reply:
ok